asreview/webapp/src/HomeComponents/DashboardComponents/ProjectTable.js

```diff
@@ -191,7 +191,7 @@ const ProjectTable = (props) => {
    * Format date and mode
    */
   const formatDate = (datetime) => {
-    let date = new Date(datetime);
+    let date = new Date(datetime.split(" ")[0]);
```

Comment:
Using `created_at_unix` instead of `datetimeCreated` might be a better fix. Unix time is easier and more efficient for storing datetime. 

https://github.com/asreview/asreview/blob/7b85eb843e17260bc3ba866be5734f170326e1e4/asreview/webapp/src/HomeComponents/DashboardComponents/ProjectTable.js#L345